New Blog about Kadalu release 0.5

aravindavk opened the pull request:
Fixes: #22
Signed-off-by: Aravinda Vishwanathapura <mail@aravindavk.in>

aravindavk committed:
New Blog about Kadalu release 0.5

Fixes: #22
Signed-off-by: Aravinda Vishwanathapura <mail@aravindavk.in>

aravindavk commented:
Do not merge!

- The actual release of container images yet to be done
- Docs link needs to be fixed
- Changelog.md not yet updated

amarts commented on _posts/2020-01-30-announcing-kadalu-0.5.md:
```diff
@@ -0,0 +1,82 @@
+title: "Announcing Kadalu Kubernetes Storage 0.5"
+author: kadalu
+layout: post
+description: "The Kadalu team is happy to announce a new version of Kadalu Kubernetes Storage, 0.5. Kadalu Kubernetes Storage is a lightweight solution to provide storage for applications running in Kubernetes."
```
s/The Kadalu team/Kadalu team/ ??

same below.

amarts reviewed (approved):
:+1:

sankarshanmukhopadhyay commented:
There's a small set of changes I'll need to make to the RN before we go live. I should be able to complete it by late afternoon (India hours)

sankarshanmukhopadhyay commented on _posts/2020-01-30-announcing-kadalu-0.5.md:
```diff
@@ -0,0 +1,82 @@
+---
+title: "Announcing Kadalu Kubernetes Storage 0.5"
```
Announcing Kadalu Storage 0.5

aravindavk replied:
done

sankarshanmukhopadhyay commented on _posts/2020-01-30-announcing-kadalu-0.5.md:
```diff
@@ -0,0 +1,82 @@
+title: "Announcing Kadalu Kubernetes Storage 0.5"
+author: kadalu
+layout: post
+description: "The Kadalu team is happy to announce a new version of Kadalu Kubernetes Storage, 0.5. Kadalu Kubernetes Storage is a lightweight solution to provide storage for applications running in Kubernetes."
```
We are excited to make available a new release of Kadalu Storage. The Kadalu Storage 0.5 continues to provide a lightweight solution to provision persistent storage for applications running in Kubernetes

aravindavk replied:
done

sankarshanmukhopadhyay commented on _posts/2020-01-30-announcing-kadalu-0.5.md:
```diff
@@ -0,0 +1,82 @@
+
+### Kadalu Kubectl plugin
+
+Kadalu Kubectl plugin was introduced during the release 0.4, which can be easily installed using `pip3 install kubectl-kadalu`. Kadalu Operator install subcommand is introduced with this release. To install Kadalu on an already running Kubernetes cluster,
```
We continue to add capabilities to make it easy to work with the Kadalu Kubectl plugin available since 0.4 release. This release has the install subcommand for the Kadalu Operator thus enabling installation of Kadalu on an already running Kubernetes cluster.

sankarshanmukhopadhyay commented on _posts/2020-01-30-announcing-kadalu-0.5.md:
```diff
@@ -0,0 +1,82 @@
+
+The example above shows adding raw device, Kadalu also supports adding the storage by specifying a path or a PVC name from other storage providers(Example Azure PV or Amazon EBS).
+
+### External Gluster Storage support
```
Support for external Gluster volume

sankarshanmukhopadhyay commented on _posts/2020-01-30-announcing-kadalu-0.5.md:
```diff
@@ -0,0 +1,82 @@
+
+More details about this feature are available [here](https://kadalu.io/docs/k8s-storage/latest/external-gluster-storage)
+
+### Gluster upgrade
```
New images with latest stable Gluster

sankarshanmukhopadhyay commented on _posts/2020-01-30-announcing-kadalu-0.5.md:
```diff
@@ -0,0 +1,82 @@
+
+## What to expect in the next release?
+
+We are happy to solve the Kubernetes storage requirements; more exciting features are coming up. Feel free to open new issues if your Cloud storage use cases are not covered.
```
We continue to make Kadalu more natively integrated with Kubernetes. There are some major features we have coming up. You can use our issue tracker for use cases and workloads which need to be reviewed for priority.

sankarshanmukhopadhyay reviewed (changes requested):
Should we introduce the way in which control plane can be switched over to Kadalu (from Heketi)? It would be good to introduce it via the blog post in the release.

aravindavk committed:
Updated the blog content

Signed-off-by: Aravinda Vishwanathapura <mail@aravindavk.in>

aravindavk commented:
Addressed the comments, @sankarshanmukhopadhyay please review.

> Should we introduce the way in which control plane can be switched over to Kadalu (from Heketi)? It would be good to introduce it via the blog post in the release.

I think we can write about it without going into detail. @amarts please suggest.

sankarshanmukhopadhyay commented:
> Addressed the comments, @sankarshanmukhopadhyay please review.
> 

LGTM. Thanks Aravinda!

> > Should we introduce the way in which control plane can be switched over to Kadalu (from Heketi)? It would be good to introduce it via the blog post in the release.
> 
> I think we can write about it without going into detail. @amarts please suggest.

Amar, can we add a small bit about this and push the blog live?

aravindavk committed:
Added details about Heketi migration

Signed-off-by: Aravinda Vishwanathapura <mail@aravindavk.in>

amarts commented on _posts/2020-01-30-announcing-kadalu-0.5.md:
````diff
@@ -0,0 +1,83 @@
+```
+
+More details about this feature are available [here](https://kadalu.io/docs/k8s-storage/latest/external-gluster-storage)
+
````
> > Amar, can we add a small bit about this and push the blog live?

I think we should say below here:

Note that, users who would like to use previously created gluster volume as a single PV (possibly with heketi or similar projects) can use this feature too. The document mentioned about provides details on how to achieve it.

aravindavk committed:
Renamed the file

Signed-off-by: Aravinda Vishwanathapura <mail@aravindavk.in>

aravindavk committed:
Updated about Heketi migration

Signed-off-by: Aravinda Vishwanathapura <mail@aravindavk.in>

sankarshanmukhopadhyay reviewed (approved):
LGTM. Please go live!
